### PR TITLE
[WFLY-8571]: Upgrade maven from 3.3.9 to 3.5.2

### PR DESCRIPTION
**Pull request by langfr**

### CODE REVIEW

**Review by ctomc — APPROVED** [review]

**Comment by bstansberry** [comment]
@Skullman -- please approve.

**Comment by bstansberry** [comment]
If we're doing this here I assume we should do it in core as well.

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -1 +1 @@
-distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.9/apache-maven-3.3.9-bin.zip
\ No newline at end of file
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```
Comment (slachiewicz) [inline .mvn/wrapper/maven-wrapper.properties]: should be 3.5.2 now

**Comment by kabir** [comment]
@Skullman @psakar Do we want this?

**Comment by psakar** [comment]
@kabir I do not know the answer yet. I've requested installation of maven 3.5.2 into build environment just today

**Comment by kabir** [comment]
@langfr As this pull request has  been held up for a long time, I am closing it to get the pull request queue under control again. Since this is very old now, it will be a very long time until this one will be addressed. Also, I don't have much time to spend on this to evaluate on a case by case basis.

If this is something important to you or the project please feel free to recreate the pull request. That way it will come on top of the pile again, and be looked at again sooner. 

If it is a new **feature**, please follow the new process for features. Ping @kabir or @jamez if you are unsure what this means, and we can help. Please do that before opening the pull request though.

Sorry for any inconvenience!

**Pull request closed without merging**